frontend/packages/core-api-client/src/lib/client/Recipient.ts

```diff
@@ -0,0 +1,31 @@
+import { BaseRESTClient } from './BaseRESTClient';
+import {Response} from "../types";
+import {
+  Recipient,
+  RecipientDefinition, RecipientGetRequest,
+  RecipientList, RecipientListRequest
+} from "../types/client/Recipient";
+
+export class RecordClient {
+  restClient: BaseRESTClient;
+
+  constructor(restClient: BaseRESTClient) {
+    this.restClient = restClient;
+  }
+
+  create = (recipient: RecipientDefinition): Promise<Response<RecipientDefinition, Recipient>> => {
+    return this.restClient.post( '/records', recipient);
+  };
+
+  list = (request: RecipientListRequest): Promise<Response<undefined, RecipientList>> => {
+    const { databaseId, formId } = request;
+    const url = `/records?databaseId=${databaseId}&formId=${formId}`;
+    return this.restClient.get(url);
+  };
+
+  get = async (request: RecipientGetRequest): Promise<Response<undefined, Recipient>> => {
```

Comment:
I'd also move these methods away from using a request object.

Either
```
list = (databaseId: string, formId: string)
get = async (databaseId: string, formId: string; recordId: string)
```
or
```
type FormLookup = {
  databaseId: string;
  formId: string;
};

type RecordLookup = {
  databaseId: string;
  formId: string;
  recordId: string;
};
list = (formLookup: FormLookup)
get = async (recordLookup: RecordLookup)
```

Reply:
I realise the second option is essentially the same as the request object, but it moves away from the abstract concept of a request to something more concrete. Request can also be confused with http request objects.

Reply:
I'll keep the object format to prevent confusion about which id is which